components/brave_wallet/browser/brave_wallet_p3a.cc

```diff
@@ -0,0 +1,69 @@
+/* Copyright 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_wallet/browser/brave_wallet_p3a.h"
+
+#include "base/metrics/histogram_macros.h"
+#include "brave/components/brave_wallet/browser/brave_wallet_service.h"
+#include "brave/components/brave_wallet/browser/keyring_service.h"
+#include "brave/components/brave_wallet/browser/pref_names.h"
+#include "components/prefs/pref_service.h"
+
+namespace brave_wallet {
+
+// Has the Wallet keyring been created?
+// i) No, ii) Yes
+void RecordKeyringCreated(mojom::KeyringInfoPtr keyring_info) {
+  const int max_bucket = 2;
+  UMA_HISTOGRAM_EXACT_LINEAR(
```

Comment:
you could use boolean histogram, just to save a couple of lines

Reply:
Thanks, updated